scripts/pop_sim.py

```diff
@@ -1,16 +1,15 @@
 from neurophotonics.sim import Tissue, Fluorescence, Detection
+from scripts.helper import keys_used
 
-try:
-    Tissue.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Tissue already populated")
-
-try:
-    Detection.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Detection already populated")
-
-try:
-    Fluorescence.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Fluorescence already populated")
+print(
+    "keys processed:",
+    keys_used(
+        Tissue.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    )
+    + keys_used(
+        Detection.populate(reserve_jobs=True, display_progress=False, processes=2)
+    )
+    + keys_used(
+        Fluorescence.populate(reserve_jobs=True, display_progress=False, processes=2)
```

Comment:
since `Fluorescence.make` already uses multiprocessing, how does multiprocessing in `populate` interact with it?

Reply:
To be perfectly honest, I am not entirely sure. My best guess is that we pay an overhead cost if a single key's make function already uses all available resources, and see an improvement to performance if some resources would have been left over.

Since I know it has previously worked while processes=1024, I am sure it will function when multi-processing is on. Choosing good values here is definitely important. However, I think it would be best to leave it as it is for now, and further tune the multiprocessing in a separate PR.